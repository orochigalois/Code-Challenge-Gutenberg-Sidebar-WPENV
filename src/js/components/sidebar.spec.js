
jest.setTimeout(30000)
import {
    createNewPost,
    enablePageDialogAccept,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

import {
    clickRadio,
    openSidebar
    
} from '../test-helpers'

const f={
    add: (n1,n2) => n1+n2
}
describe( 'ðŸ”¬ Ad Sideber', () => {
    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    test( 'ðŸ”¬ Advertisements sidebar should be available', async () => {
        expect(f.add(2,2)).toBe(4);
        await openSidebar()
		// expect( await getEditedPostContent() ).toMatchSnapshot();
    } )

} );
