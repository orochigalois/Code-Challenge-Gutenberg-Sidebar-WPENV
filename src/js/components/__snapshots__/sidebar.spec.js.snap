// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ğŸ”¬ Ad Sideber ğŸ”¬ Advertisements sidebar should be available 1`] = `""`;
